docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,398 @@
+these privileges, see <<privileges-list-indices>>. 
+
+Let's assign the `kibana_user` role to your user. Go back to the 
+*Management / Security / Users* page and select your user. Add the `kibana_user` 
+role and save the change. For example:
+
+[role="screenshot"]
+image::security/images/assign-role.jpg["Assigning a role to a user in Kibana"]
+
+This user now has the minimum privileges required to use {kib}. 
+
+If you completed all of the steps in <<get-started-elastic-stack>>, you should 
+have {metricbeat} data stored in {es}. Let's create two roles that grant 
+different levels of access to that data. 
+
+Go to the *Management / Security / Roles* page and click *Create role*. 
+
+Create a `metricbeat_reader` role that has `read` and `view_index_metadata` 
+privileges on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-reader-role.jpg["Creating a role in Kibana"]
+
+Create a `metricbeat_writer` role that has `manage_index_templates` and `monitor` 
+cluster privileges, as well as `write`, `delete`, and `create_index` privileges
+on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-writer-role.jpg["Creating another role in Kibana"]
+
+Now go back to the *Management / Security / Users* page and assign these roles 
+to the appropriate users. Assign the `metricbeat_reader` role to your personal 
+user.  Assign the `metricbeat_writer` role to the `logstash_internal` user.
+
+The list of users should now contain all of the built-in users as well as the 
+two you created. It should also show the appropriate roles for your users:
+
+[role="screenshot"]
+image::security/images/management-users.jpg["User management screenshot in Kibana"]
+
+If you want to learn more about authorization and roles, see <<authorization>>.
+
+[float]
+[[gs-logstash-security]]
+=== Add user information in {ls}
+
+In order for {ls} to send data successfully to {es}, you must configure its 
+authentication credentials in the {ls} configuration file.  For example, update 
+the `demo-metrics-pipeline.conf` file in your {ls} directory:
+
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+filter {
+  if [system][process] {
+    if [system][process][cmdline] {
+      grok {
+        match => {
+          "[system][process][cmdline]" => "^%{PATH:[system][process][cmdline_path]}"
+        }
+        remove_field => "[system][process][cmdline]"
+      }
+    }
+  }
+}
+
+output {
+  elasticsearch {
+    hosts => "localhost:9200"
+    manage_template => false
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
+    user => "logstash_internal" <1>
+    password => "your_password" <2>
+  }
+}
+----
+<1> Specify the `logstash_internal` user that you created earlier in this tutorial. 
+<2> Specify the password that you chose for this user ID.
+
+If you prefer not to put your user ID and password in the configuration file, 
```

Comment:
I really went down a rabbit hole because I copy/pasted your first config example and then decided that I wanted to use the keystore instead. So I modified my config, but forgot to add the $ (even though I know it's  required). I wasted time recreating the keystore (I thought that I might have made a typo...there's no way to know for sure because when you add a key to the keystore, it doesn't let you confirm the value) .

I might have avoided this particular rabbit hole  if I'd gone directly to the keystore config example first and copied that. 

You might want to reformat this content or add something to intro so that careless readers (like me!) will know up front that they can use a keystore. Might be worth even pointing out the dollar sign.  This was frustrating because it was one of those dumb mistakes that your eyes skip over and the error message is not useful.